azkaban-web-server/src/main/java/azkaban/imagemgmt/models/ImageOwnership.java

```diff
@@ -0,0 +1,30 @@
+package azkaban.imagemgmt.models;
+
+public class ImageOwnership extends BaseModel {
+  private String owner;
+  private String role;
```

Comment:
Can you please use enum for role over here?

Reply:
Done